docs/adding_software/overview.md

```diff
@@ -4,17 +4,39 @@ We welcome contributions to the EESSI software stack. This page shows the proced
 
 ## Contribute a software to the EESSI software stack
 
-<p align="center">
-  <img width="100%" src="../../img/contribute_software_workflow.png" alt="EESSI contribute software">
-</p>
+<pre class="mermaid">
+%%{init: { 'theme':'forest', 'sequence': {'useMaxWidth':false} } }%%
+flowchart TB
+    I(contributor)  
+    K(reviewer)
+    A(Is there an EasyConfig for software) -->|No|B(Create an EasyConfig and contribute it to EasyBuild)
+    A --> |Yes|D(Create a PR to software-layer)
+    B --> C(Evaluate and merge pull request)
+    C --> D
+    D --> E(Review PR & trigger builds)
+    E --> F(Debug build issue if needed)
+    F --> G(Deploy tarballs to S3 bucket)
+    G --> H(Ingest tarballs in EESSI by merging staging PRs)
+     classDef blue fill:#9abcff,stroke:#333,stroke-width:2px;
+     class A,B,D,F,I blue
+</pre>
+
 
 ## Contributing a ReFrame test to the EESSI test suite
 
 Ideally, a contributor prepares a ReFrame test for the software to be added to the EESSI software stack. 
 
-<p align="center">
-  <img width="100%" src="../../img/contribute_test_workflow.png" alt="EESSI contribute test">
-</p>
+<pre class="mermaid">
```

Comment:
Mermaid automatically does a left aligned. So the two graphs are not that well aligned because of the contributor and reviewer note in the graph above. If the second graph is centred they align much more.    
```suggestion
<div style="display:flex;justify-content:center;">
<pre class="mermaid">
```